src/icons/secure-lock.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g id="SF - Secure" opacity="0.75">
```

Comment:
The opacity of the icon should be applied within the CSS styles. And the group definition can be removed.